frontend/pages/admin/IntegrationsPage/IntegrationsPage.tsx

```diff
@@ -82,12 +80,8 @@ const IntegrationsPage = ({
   );
 
   if (!appConfig) return <></>;
-  const isManagedCloud = appConfig.license.managed_cloud;
-  if (section?.includes("conditional-access") && !isManagedCloud) {
-    router.push(paths.ADMIN_SETTINGS);
-  }
 
-  const navItems = getIntegrationSettingsNavItems(isManagedCloud);
+  const navItems = getIntegrationSettingsNavItems();
```

Comment:
Always include conditional access nav item - gating will be at a different layer